pages/upgrade/graylog-2.2.rst

```diff
@@ -0,0 +1,79 @@
+**************************
+Upgrading to Graylog 2.2.x
+**************************
+
+.. _upgrade-from-21-to-22:
+
+This file only contains the upgrade note for the upcoming release.
+Please see `our documentation <http://docs.graylog.org/en/latest/pages/upgrade.html>`_
+for the complete upgrade notes.
+
+Email Alarm Callback
+====================
+
+In previous Graylog versions, creating an alert condition on a stream and adding an alert receiver meant that, if no alarm callback existed for that stream, Graylog would use an Email alarm callback by default.
```

Comment:
Pretty complicated sentence. Maybe simplify like: In previous Graylog versions an Email alarm callback was created by default when no callback existed for a stream.